### PR TITLE
feat(blog): add integrating with gatsby blog post

### PR DESCRIPTION
**Pull request by calcsam**
Adds an integrating with Gatsby blog post.

### CODE REVIEW

File: docs/blog/2019-04-02-integrating-with-gatsby/index.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+Typically, there are four key phases or stages of partnership, depending on at any given time how much it makes sense for us to work together.
+
+These stages are:
+
+- **First,** your team building a Gatsby integration
+
+- **Second,** your team launching the integration
+
+- **Third, **co-marketing, and
```
Comment (muescha) [inline docs/blog/2019-04-02-integrating-with-gatsby/index.md]:
```suggestion
- **Third,** co-marketing, and
```

File: docs/blog/2019-04-02-integrating-with-gatsby/index.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+In terms of what a "first-class" Gatsby Preview integration and partnership looks like, there are several components:
+
+#### Engineering:
+
+- A staging environment accessible by API. Some examples: [Contentful’s Preview API](https://www.contentful.com/developers/docs/references/content-preview-api/), [DatoCMS staging environment](https://www.datocms.com/changelog/multiple-deployment-environments), etc
+
+- Webhooks firing if, and only if, the staging content is updated. These webhooks should fire in less than 3 seconds after the content changes.
+
+- A button in the CMS editor where users can click on the button
```
Comment (KyleAMathews) [inline docs/blog/2019-04-02-integrating-with-gatsby/index.md]: Click through to Preview

File: docs/blog/2019-04-02-integrating-with-gatsby/index.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+We’re in the process of opening it to more users and building support in for additional CMS systems. If this is interesting to you, we’d love to chat more.
+
+In terms of what a "first-class" Gatsby Preview integration and partnership looks like, there are several components:
+
+#### Engineering:
+
+- A staging environment accessible by API. Some examples: [Contentful’s Preview API](https://www.contentful.com/developers/docs/references/content-preview-api/), [DatoCMS staging environment](https://www.datocms.com/changelog/multiple-deployment-environments), etc
+
+- Webhooks firing if, and only if, the staging content is updated. These webhooks should fire in less than 3 seconds after the content changes.
```
Comment (KyleAMathews) [inline docs/blog/2019-04-02-integrating-with-gatsby/index.md]: "Only if" isn't necessary

File: docs/blog/2019-04-02-integrating-with-gatsby/index.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+- Coordination and communication between our team and your marketing team. Executive buy-in or support on your end for co-marketing.
+
+- Co-marketing, such as co-branded campaigns or content discussing [YOUR SOLUTION] + Gatsby Preview.
+
+- Our team working to train your sales team (both account executives and sales engineers) on describing the benefits of Gatsby to prospective customers. If relevant, we can work to train your customer success team as well.
+
+## Conclusion: The Content Mesh
+
+One of the trends we’re seeing is that the CMS is becoming a "[content mesh](https://www.gatsbyjs.org/blog/2018-10-04-journey-to-the-content-mesh/)". It’s unbundling, and best of breed tools are taking over separate pieces of functionality (headless CMS, specialized CMS, forms, analytics, auth, A/B testing, UI frameworks)....
```
Comment (KyleAMathews) [inline docs/blog/2019-04-02-integrating-with-gatsby/index.md]: Unbundling what?

**Comment by calcsam** [comment]
Thanks @KyleAMathews. Lint fail isn't this code -- going to merge in.

**Pull request merged**